plugins/zoom/plugin.spec.yaml

```
@@ -20,18 +20,24 @@ resources:
 enable_cache: false
 
 connection:
-  api_key:
-    title: API Key
-    description: JWT API Key
+  client_id:
+    title: Client ID
+    description: Zoom app client ID
     type: credential_secret_key
     required: true
-    example: 'abcdefgABCDEFG12345678'
-  secret:
-    title: Secret
-    description: JWT Secret
+    example: '41tCqP_VZ5jLeaPcBmIrN'
```

https://docs.rapid7.com/insightconnect/style-guide/#input-examples

```suggestion
    example: "9de5069c5afe602b2ea0a04b66beb2c0"
```